<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="ControlGroup">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Speed Control</string>
    </property>
    <widget class="QCheckBox" name="CruiseControlEnable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>78</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="CruiseSpeed">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ControlGroup_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Hill Decent</string>
    </property>
    <widget class="QCheckBox" name="HillControlEnable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>78</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="HillSpeed">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>90</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSlider" name="HillSpeedSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>25</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="NavigationGroup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Auto Nav</string>
    </property>
    <widget class="QComboBox" name="WaypointSelection">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TargetHeading">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="DisplayBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Display Box</string>
    </property>
    <widget class="QLCDNumber" name="SpeedDisplay">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="FullStop">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Deactivate">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Deactivate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Activate">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Activate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
